### PR TITLE
Adding colormap to controlled terms

### PR DESCRIPTION
**Pull request by lzehl**
+ adding Colormap schema with category "color"
+ adding colormap terminology instance to Terminology instances
+ adding first selection of Colormap instances (all matplotlib colormaps)

Naming convention for Colormap instances:
`»package«.»module«.»colormap-name«`
Examples:
```
matplotlib.colormaps.viridis
mathworks.colormap.winter
```

Note: I kept the definition of the colormaps superficial. I suggest we can provide more details on the colormaps later on.
This PR is related to https://github.com/HumanBrainProject/openMINDS_SANDS/pull/237

### CODE REVIEW

File: instances/matplotlib.colormaps.Greys.jsonld [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (aeidi89) [inline instances/matplotlib.colormaps.Greys.jsonld]: This instance seems to be a duplicate.
With the exception of the jsonld name, all the information is identical.

- instances/colormap/matplotlib.colormaps.Greys.jsonld (keep this instance, above)

- instances/matplotlib.colormaps.Greys.jsonld (delete this instnace)

**Review by aeidi89 — CHANGES_REQUESTED** [review]
I fixed a few minor typos (direct commit to this branch).
I found one instance that I think is a duplicate of another. I've made a comment that needs a review.

Otherwise it looks good.

@lzehl could you have a look?

File: instances/matplotlib.colormaps.Greys.jsonld (hunk 1 above)
Comment (lzehl) [reply instances/matplotlib.colormaps.Greys.jsonld]: deleted. I wondered where this file ended up... the computer froze and then it was gone... I did not realize that I actually saved it in the wrong place :see_no_evil:

**Comment by lzehl** [comment]
@aeidi89 thanks for the corrections. final issue (file duplicate) is deleted. Feel free to merge if everything else is okay now

**Pull request merged**